client/src/sections/resources/Banishers.tsx

```diff
@@ -0,0 +1,175 @@
+    this.access = access;
+    this.banishType = banishType;
+    this.banishedMonster = banishedMonster ?? "";
+    this.href = href ?? "";
+    this.isEquipped = isEquipped ?? true;
+    this.equipItem = equipItem ?? "";
+    this.extraString = extraString ?? "";
+  }
+
+  // Returns if the banish is accessible for filtering.
+  accessible(): boolean {
+    return this.access && this.numberRemaining > 0;
+  }
+
+  // Renders the tile. Switches the line's stylings based on banishType.
+  render(): React.ReactNode {
+    const possibleStrings: { [key: string]: string } = {
+      skill: `${plural(this.numberRemaining, "cast")} of ${this.name}`,
+      item: `${plural(this.numberRemaining, "use")} of ${this.name}`,
+      equip: `${plural(this.numberRemaining, "use")} of ${this.name}${
+        !this.isEquipped ? ` (equip your ${this.equipItem}!)` : ""
+      }`,
+    };
+
+    return <Line href={this.href}>{possibleStrings[this.banishType]}</Line>;
+  }
+}
+
+// TO-DO LIST FOR THIS TILE:
+//   -  We could manually construct a map of zones where banishing monsters is desirable and filter out each monster you don't want to banish, then check which monsters are banished against that list
+
+const Banishers: React.FC = () => {
+  // List out all banishes in BanisherObject construction.
+  const allBanishers = [
+    new BanisherObject(
+      "Feel Hatred",
+      3 - useGet("_feelHatredUsed"),
+      useHave($skill`Emotionally Chipped`),
+      "skill"
+    ),
+    // Should we add an extrastring w/ # of banishes used today? Would be cute.
+    new BanisherObject(
+      "Bowl a Curveball",
+      1,
+      useHave($item`cosmic bowling ball`),
+      "item"
+    ),
+    new BanisherObject(
+      "Show Your Boring Familiar Pictures",
+      Math.floor(useGet("scrapbookCharges") / 100),
+      useHave($item`familiar scrapbook`),
+      "equip",
+      "/inventory.php?which=2",
+      undefined,
+      useHaveEquipped($item`familiar scrapbook`),
+      "familiar scrapbook"
+    ),
+    new BanisherObject(
+      "Show Them Your Ring",
+      +!useGet("_mafiaMiddleFingerRingUsed"),
+      useHave($item`mafia middle finger ring`),
+      "equip",
+      "/inventory.php?which=2",
+      undefined,
+      useHaveEquipped($item`mafia middle finger ring`),
+      "mafia middle finger ring"
+    ),
+  ];
+
+  // Render the possible banishers via a map of our banisher listing.
+  const renderedBanishes: React.ReactNode[] = allBanishers
+    .filter((banish) => banish.accessible())
+    .map((banish) => banish.render());
+
+  return (
+    <Tile
+      header="Banishes"
+      imageUrl="/images/itemimages/Bowlcurve.gif"
+      hide={renderedBanishes.length < 1}
+    >
+      {renderedBanishes}
+    </Tile>
+  );
+};
+
+export const BanishedMonsters: React.FC = () => {
+  // Snag the banished monsters. Format is:
+  //   [MONSTER]:[BANISHER]:[TURNBANISHED]
+  // (That format sucks, dude.)
+  const _banishedMonsters = useGet("banishedMonsters");
```

Comment:
```suggestion
  const _banishedMonsters = useGet("banishedMonsters") ?? 0;
```
banishedMonsters is empty by default, it'll end up being undefined and `.split()` won't be happy